demos/browser/ChimeMeetings.js

```diff
@@ -0,0 +1,49 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+const AWS = require('aws-sdk');
+
+module.exports = class ChimeMeetings {
+
+    // Create an AWS SDK Chime object. Region 'us-east-1' is currently required.
+    // Use the MediaRegion property below in CreateMeeting to select the region
+    // the meeting is hosted in.
+    
+
+    constructor(currentRegion, controlRegion, endpoint, useChimeGlobal){
+        console.log("const controlRegion",controlRegion);
+        this.chime = new AWS.Chime({ region: currentRegion });
+
+        // Set the AWS SDK Chime endpoint. The global endpoint is https://service.chime.aws.amazon.com
+        this.chime.endpoint = new AWS.Endpoint(endpoint);
+        this.controlRegion = (!!controlRegion) ? currentRegion : controlRegion;
+        console.log("Const this.controlRegion",this.controlRegion);
+        this.chimeMeetings = new AWS.ChimeSDKMeetings({ region: this.controlRegion});
```

Comment:
So do we need to test again global SDK or can we just test again the meeting SDK? Its a bit confusing to do it like this since its a demo so if a builder looks at this its confusing :)

Reply:
We want to continue to test the Chime namespace Meeting APIs (since they still exist and are available for public use) but we want it to be the non-default case.